pallets/subtensor/src/utils/identity.rs

```diff
@@ -106,4 +106,32 @@ impl<T: Config> Pallet<T> {
             && identity.description.len() <= 1024
             && identity.additional.len() <= 1024
     }
+
+    /// Swaps the hotkey of a delegate identity from an old account ID to a new account ID.
+    ///
+    /// # Parameters
+    /// - `old_hotkey`: A reference to the current account ID (old hotkey) of the delegate identity.
+    /// - `new_hotkey`: A reference to the new account ID (new hotkey) to be assigned to the delegate identity.
+    ///
+    /// # Returns
+    /// - `DispatchResult`: Returns `Ok(())` if the swap is successful. Returns an error variant of `DispatchResult` otherwise.
+    pub fn swap_delegate_identity_coldkey(
+        old_coldkey: &T::AccountId,
+        new_coldkey: &T::AccountId,
+    ) -> DispatchResult {
```

Comment:
I understand the intuition behind returning a DispatchResult here , but we dont return errors for any of the other swap steps. 

This is because the top level ensures should catch all these